run.py

```diff
@@ -0,0 +1,153 @@
+import argparse
+import csv
+from pathlib import Path
+import os
+import pickle
+import yaml
+
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer
+from bertopic.vectorizers import ClassTfidfTransformer
+from spacy.lang.en import stop_words
+from sentence_transformers import SentenceTransformer
+
+from PatientX.models.BERTopicModel import BERTopicModel
+from PatientX.MistralRepresentation import MistralRepresentation
+
+
+def read_data(datafolder: Path):
+    """
+    Read in data
+
+    datafolder: Path
+    Returns: List containing documents
+    """
+    dfs = []
+
+    assert(os.path.isdir(datafolder))
+
+    for filename in os.listdir(datafolder):
+        filepath = os.path.join(datafolder, filename)
+        if os.path.isfile(filepath):
+            try:
+                df = pd.read_csv(filepath)
+                dfs.append(df.copy(deep=True))
+            except csv.Error as e:
+                pass
+
+    full_dataset = pd.concat(dfs, ignore_index=True)
+    cleaned_text = pd.DataFrame()
+    try:
+        grouped_dataset = full_dataset.groupby(['forum', 'thread_title', 'message_nr'], as_index=False).agg(
+            {'post_message': ''.join})
+
+        grouped_dataset['post_message'] = grouped_dataset['post_message'].str.strip().replace(r'\n', ' ', regex=True)
+        cleaned_text = grouped_dataset['post_message'].replace({r'\s+$': '', r'^\s+': ''}, regex=True).replace(r'\n', ' ',
+                                                                                                           regex=True)
+        cleaned_text = cleaned_text.replace(r'\t', ' ', regex=True)
+        cleaned_text = cleaned_text.replace(r'\r', ' ', regex=True)
+    except KeyError as e:
+        raise KeyError("Check README file for proper data format")
+
+    return cleaned_text.tolist()
+
+
+def get_count_vectorizer():
+    custom_stop_words = list(stop_words.STOP_WORDS)
```

Comment:
I'm guessing that some of the words added in the next line are already a part of this list. Consider making this a set while adding stop works and then converting the set to a list when passing it as an argument when CountVectorizer is used below.